src/components/ButtonPanel.js

```diff
@@ -1,10 +1,39 @@
 import React from 'react';
+import Button from './Button';
 import './ButtonPanel.css';
 
 function ButtonPanel() {
   return (
     <div className="ButtonPanel" id="button-panel">
-      <p> Calculator App: Hello world!</p>
+      <div className="group1">
```

Comment:
- [ ] [OPTIONAL] Think of a way to avoid repeating the code so much. What kind of map could you make in order to avoid repeating the Button element for each operation, or inside each div?

Reply:
A map inside a for loop?
A for loop for the divs and a map for 5 different arrays.
For example: 
Array 1:
['AC','+/-', etc]
Array 2:
[7,8,9,X]
....
etc